app/src/main/java/org/sopt/and/feature/login/LogInScreen.kt

```diff
@@ -0,0 +1,199 @@
+package org.sopt.and.feature.login
+
+import android.widget.Toast
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.LocalLifecycleOwner
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.flowWithLifecycle
+import org.sopt.and.designsystem.component.textfield.OnboardingTextField
+import org.sopt.and.designsystem.theme.ANDANDROIDTheme
+
+@Composable
+fun LogInRoute(
+    modifier: Modifier = Modifier,
+    navigateToSignUp: () -> Unit,
+    navigateToMain: (String) -> Unit,
+    viewModel: LogInViewModel = hiltViewModel(),
```

Comment:
modifier는 모든 옵셔널 파라미터 중에 최상단으로 와야해요.
이 순서 지켜주세용
- 필수 
- 옵셔널(모디파이어 최상단)

```suggestion
    navigateToSignUp: () -> Unit,
    navigateToMain: (String) -> Unit,
    modifier: Modifier = Modifier,
    viewModel: LogInViewModel = hiltViewModel(),
```